drivers/ethernet/Kconfig.mcux

```diff
@@ -36,6 +38,14 @@ config ETH_MCUX_RMII_EXT_CLK
 	  Setting this option will configure MCUX clock block to feed RMII
 	  reference clock from external source (ENET_1588_CLKIN)
 
+config ETH_MCUX_REFCLK_50MHZ
+	bool "Configure PHY Refclk Crystal is 50MHz (else 25MHz)"
+	default n if ETH_MCUX_RMII_EXT_CLK
+	default y
+	help
+	  Setting this option will configure the PHY for a 50MHz refclk.
```

Comment:
This belongs in device tree instead of kconfig IMO, and `ETH_MCUX_RMII_EXT_CLK` probably as well.